src/pages/design/icons/library.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Icon Library
+---
+
+import { graphql } from 'gatsby';
+
+<!-- ### Tokens
+
+Icon tokens represent specific concepts in Zendesk products.
+Before adding an icon to your project, check the list below to see if the concept has an icon tokenassociated with it.
+To suggest a new token, please take a look at our [icon token decision tree](https://miro.com/app/board/uXjVOcPRJo0)
+and fill out a Jira template if your request meets the criteria for addition. -->
+
+### All Icons
+
+This is a list of all the icons in the Garden library, independent of context.
+Use these icons only if you can’t find a suitable token.
+
+import { SvgSearch } from '../../../examples/design/icons/SvgSearch';
+
+<SvgSearch data={props.data.svgIcons} />
+
+<!-- ### Contributing a token
+
+To suggest a new token, please consider the following:
+
+- Is my token repeatable? If the action will only appear once or very few times, using a one-off icon is fine.
+
+- Will it be used across multiple products?
+
+- Can it be consistently tied with a concept? Is there a clear action?
+
+If you can answer “yes” to these questions, please post in #garden-seeds with the header “Token Request” and the Garden team will get back to you! -->
+
+### Contributing an icon
+
+If none of the current Garden icons will work for your token, you can contribute a new icon.
+
+To suggest a new token, please consider the following:
+
+- Is my icon repeatable? Can it be used in at least 3 places?
+  If not, consider using a local icon rather than adding to the Garden library.
+
+- Will it be used across mutliple products?
+
+- Is it unique? Is it unlike any exisiting Garden icon?
+
+To suggest an icon, please post in [#garden-seeds](https://app.slack.com/client/T024F4EL1/C02JQBLQX5W/)
```

Comment:
Much of this can be housed internally in my experience but that's if the design system team has good relations with product teams and solid communication channels. Imo the website taking on this burden more since we don't have that culture as of now.

Reply:
Sorry, the language about this process is out of date definitely needs to be updated. That said, I think having some reference to our internal [Confluence page](https://zendesk.atlassian.net/wiki/spaces/GARDEN/pages/5379294927/Making+Garden+icons+revised+Q1+2022) is vital here. We can't let the open source nature of this project disadvantage our home team.